docs/release-playbook.md

```diff
@@ -21,125 +21,58 @@ release process.
-- [ ] [Create draft release announcement](https://docs.google.com/document/d/1wDvulLlj4NAlPZamdlEVFORks3YXJonCjyuQMUQEmB0/edit)
-- [ ] Send for review the release announcement PR:
-- [ ] Push the release, notify package maintainers:
-- [ ] Update the documentation
-- [ ] Push the blog post
-- [ ] Update the [release page](https://github.com/bazelbuild/bazel/releases/)
-```
-
-Keep the task list updated and check boxes as you follow the release process. [Example](https://github.com/bazelbuild/bazel/issues/7816#issuecomment-497037232).
-
-## Create a Candidate
-
-Create candidates with the `release.sh` script. See above how to pick a good baseline commit.
-
-*  If it's the first candidate for this version, run:
-
-    ```bash
-    RELEASE_NUMBER=<CURRENT RELEASE NUMBER x.yy.z>
-    BASELINE_COMMIT=01234567890abcdef # From the Setup phase
-    git clone https://github.com/bazelbuild/bazel.git ~/bazel-release-$RELEASE_NUMBER
-    cd ~/bazel-release-$RELEASE_NUMBER
-    scripts/release/release.sh create $RELEASE_NUMBER $BASELINE_COMMIT [CHERRY_PICKS...]
-    ```
-
-    Note that the three-digit version is important: "0.19.0". not "0.19".
-
-*  For cherry-picks, you need `--force_rc=N` where `N` is the number of the
-    release candidate of `$RELEASE_NUMBER`. For example, the first time you do a
-    cherry-pick (after the initial candidate), N will be 2.
-
-    ```bash
-    scripts/release/release.sh create --force_rc=2 $RELEASE_NUMBER $BASELINE_COMMIT [CHERRY_PICKS...]
-    ```
-
-*  If you already did some cherry-picks and you want to add more, use "git log"
-    to find the latest commit (this corresponds to the last cherry-pick commit).
-    Use that as the new baseline and list the new cherry-picks to add on top. Or
-    simply re-use the same baseline and cherrypicks from the previous candidate,
-    and add the new cherrypicks.
-
-    ```bash
-    scripts/release/release.sh create --force_rc=3 $RELEASE_NUMBER NEW_BASELINE_COMMIT [NEW_CHERRY_PICKS...]
-    ```
-
-Resolve conflicts if there are any, type `exit` when you are done, then the script will continue.
-WARNING: `release.sh create` handles conflicts in a subshell (which is why you need to type `exit`).
-
-Editing the release notes is not needed (it will be done later).
-
-## Push the candidate
-
-1.  Run `release.sh push`. This uploads the candidate and starts the release
-    process on BuildKite.
-
-    ```bash
-    scripts/release/release.sh push
-    ```
-
-1.  Update GitHub issue with the command that was run and the new candidate name
-    (ie, 0.19.1rc3). Update the issue with the estimated release date, assuming no
-    regression is found.
+1.  Create a branch (via the GitHub UI or via the command line) named `release-X.Y.ZrcN` at the appropriate commit.
+    *   This branch is essentially used as a tag (i.e. no further commits should happen on this branch). All cherry-pick commits should continue to go into the release branch (i.e. `release-X.Y.Z`, without the `rcN` part).
```

Comment:
Is there a reason it's not a tag? There has been lots of confusion over where to "find" rc builds, as they are the only build that bazelisk can find, but a corresponding tag doesn't exist for.

Reply:
I think it's simply because it's how the pipeline is working so far. Also you can easily create a branch on the GitHub UI, but not so easily create a tag (without creating a release, that is). I can look into switching to using tags, but that'd likely be very low priority.

Anyhow, now the RC branches don't actually get deleted anymore, so "release-5.1.1rc1" *will* reliably point to the RC1 commit.

Reply:
That's true. As I said though, I'll probably look into using tags later, instead of with this PR.